netlify/templates
# Add Netlify Next.js Blog theme template

## Summary

This adds our first template out of a set of templates that we've been making at Netlify.

### Test Plan

- Open the Deploy Preview
- Verify that the new template `Bejamas X Netlify - Next.js Blog Starter` is within the index page
- Ensure that a user can click on the page and get more details
- Make sure the DTN redirects properly

### Description for changelog

Add Netlify Next.js Blog theme template
